### PR TITLE
Martial Arts Mastery

### PR DESCRIPTION
**Pull request by Geevies**
* It is now possible to know more than one martial art at the same time, while reaping the benefits of all of them.
* Only the unarmed attacks of your primary martial art will be executed, if you have more than one art, the option to choose will appear in the abilities tab.
* Created a new Martial Arts uplink category, added Sleeping Carp, 12 TC, and Swordsmanship, 5 TC, to it.
* Martial Art projectile deflection has been reflavoured to make it dodging, for the sake of believability. Ha ha ha.
* Sleeping carp now properly only instantly kills someone who's in crit.
* The Recall Teachings ability is now put into the abilities tab instead of a unique tab for the martial arts.

### CODE REVIEW

**Comment by Geevies** [comment]
!review

**Review by Alberyk — APPROVED** [review]

File: code/__defines/misc.dm [hunk 1]
```diff
@@ -217,6 +217,7 @@
 // Special return values from bullet_act(). Positive return values are already used to indicate the blocked level of the projectile.
 #define PROJECTILE_CONTINUE   -1 //if the projectile should continue flying after calling bullet_act()
 #define PROJECTILE_FORCE_MISS -2 //if the projectile should treat the attack as a miss (suppresses attack and admin logs) - only applies to mobs.
+#define PROJECTILE_DODGED     -3
```
Comment (Arrow768) [inline code/__defines/misc.dm:220]: Add a comment here to explain the difference between continue, force_miss and dodge

File: code/modules/martial_arts/martial.dm [hunk 2]
```diff
@@ -136,24 +135,28 @@
...
 		H.verbs += help_verb
-	if(make_temporary)
-		temporary = 1
-	if(temporary)
-		if(H.martial_art)
-			base = H.martial_art.base
-	else
-		base = src
-	H.martial_art = src
+		to_chat(H, SPAN_NOTICE("You can review the combos by recalling the teaches of this art in your abilities tab."))
```
Comment (Arrow768) [inline code/modules/martial_arts/martial.dm:141]:
```suggestion
		to_chat(H, SPAN_NOTICE("You can review the combos by recalling the teachings of this art in your abilities tab."))
```

**Review by Arrow768 — CHANGES_REQUESTED** [review]

**Comment by Geevies** [comment]
!review

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**